src/collections/_documentation/platforms/javascript/react.md

```diff
@@ -24,29 +24,28 @@ import * as Sentry from '@sentry/browser';
 //  dsn: "___PUBLIC_DSN___"
 // });
 // should have been called before using it here
-// ideally before even rendering your react app 
+// ideally before even rendering your react app
 
 class ExampleBoundary extends Component {
     constructor(props) {
         super(props);
-        this.state = { error: null };
+        this.state = { error: null, eventId: null };
     }
 
     componentDidCatch(error, errorInfo) {
       this.setState({ error });
       Sentry.withScope(scope => {
-        Object.keys(errorInfo).forEach(key => {
-          scope.setExtra(key, errorInfo[key]);
-        });
-        Sentry.captureException(error);
+          scope.setExtras(errorInfo);
```

Comment:
Since 5.0 is out now, switched to the cleaner syntax.

Reply:
Sweet 👍